### ISSUE TITLE
Create vm with CDI imported image failed

### ISSUE THREAD

**Issue by shiywang** [kubevirt/kubevirt#1096]
I tried to use CDI import vm images, the CDI all success
```
[root@qe-shiywang-master-etcd-1 ~]# oc get pvc
NAME         STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
golden-pvc   Bound     pvc-20b57db8-64a7-11e8-a2a8-42010af0000c   10Gi       RWO            kubevirt       9m
[root@qe-shiywang-master-etcd-1 ~]# oc get pv
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                      STORAGECLASS   REASON    AGE
pvc-20b57db8-64a7-11e8-a2a8-42010af0000c   10Gi       RWO            Delete           Bound     golden-images/golden-pvc   kubevirt                 9m
[root@qe-shiywang-master-etcd-1 ~]# oc get pod
```
then I tried to create a vm.yaml here

`oc create -f vm.yaml`
```
apiVersion: kubevirt.io/v1alpha1
kind: VirtualMachine
metadata:
  name: test-vm
spec:
  terminationGracePeriodSeconds: 0
  domain:
    resources:
      requests:
        memory: 64M
    devices:
      disks:
      - name: mydisk
        volumeName: pvcvolume
        disk:
          dev: vda
  volumes:
    - name: pvcvolume
      persistentVolumeClaim:
        claimName: golden-pvc
```

the images I used here:
http://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img
https://github.com/kubevirt/containerized-data-importer/blob/master/test/images/tinyCore.iso

then the virt-launcher log here like this
```
[root@qe-shiywang-master-etcd-1 ~]# oc logs -f virt-launcher-test-vm-782rs
mknod: missing operand after '10'
Try 'mknod --help' for more information.
chown: cannot access '/dev/kvm': No such file or directory
chmod: cannot access '/dev/kvm': No such file or directory
level=info timestamp=2018-05-31T07:51:18.383423Z pos=virt-launcher.go:141 component=virt-launcher msg="Watchdog file created at /var/run/kubevirt/watchdog-files/golden-images_test-vm"
level=info timestamp=2018-05-31T07:51:18.389892Z pos=client.go:152 component=virt-launcher msg="Registered libvirt event notify callback"
level=info timestamp=2018-05-31T07:51:18.390056Z pos=virt-launcher.go:59 component=virt-launcher msg="Marked as ready"
level=info timestamp=2018-05-31T07:51:18.390167Z pos=monitor.go:253 component=virt-launcher msg="Monitoring loop: rate 1s start timeout 5m0s"
level=info timestamp=2018-05-31T07:51:19.393202Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=error timestamp=2018-05-31T07:51:19.649800Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:19.711830Z pos=network.go:151 component=virt-launcher msg="Updated Mac for iface: eth0 - d6:cc:fe:4d:72:2b"
level=error timestamp=2018-05-31T07:51:19.726039Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:19.726145Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:19.727196Z pos=network.go:198 component=virt-launcher msg="Found nameservers in /etc/resolv.conf: \n\ufffd\u0000\r"
level=info timestamp=2018-05-31T07:51:19.727223Z pos=network.go:199 component=virt-launcher msg="Found search domains in /etc/resolv.conf: golden-images.svc.cluster.local svc.cluster.local cluster.local c.openshift-gce-devel.internal google.internal"
level=info timestamp=2018-05-31T07:51:19.727237Z pos=dhcp.go:58 component=virt-launcher msg="Starting SingleClientDHCPServer"
level=error timestamp=2018-05-31T07:51:19.779113Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:19.780434Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:19.780483Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Failed to sync vm"
level=error timestamp=2018-05-31T07:51:19.787611Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:19.788748Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:19.788798Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Failed to sync vm"
level=error timestamp=2018-05-31T07:51:19.810052Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:19.811272Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:19.811316Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=error timestamp=2018-05-31T07:51:19.852353Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:19.853516Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:19.853557Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Failed to sync vm"
level=error timestamp=2018-05-31T07:51:19.934608Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:19.935872Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:19.935935Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=error timestamp=2018-05-31T07:51:20.099339Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:20.100813Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:20.100863Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:20.409567Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=error timestamp=2018-05-31T07:51:20.428408Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:20.510423Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:20.510509Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=error timestamp=2018-05-31T07:51:21.159994Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:21.161174Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:21.161225Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:21.392193Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:22.391977Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=error timestamp=2018-05-31T07:51:22.442313Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:22.443603Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:22.443659Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=0, Domain=0, Message='Missing error')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:23.391966Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:24.391864Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=error timestamp=2018-05-31T07:51:25.005325Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:25.006805Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:25.006880Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:25.392543Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:26.392022Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:27.391914Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:28.391847Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:29.392212Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=error timestamp=2018-05-31T07:51:30.128193Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:30.129460Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:30.129506Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:30.392208Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:31.392039Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:32.393302Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:33.392021Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:34.392568Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:35.391965Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:36.392140Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:37.392405Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:38.392037Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:39.392069Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=error timestamp=2018-05-31T07:51:40.370777Z pos=converter.go:75 component=virt-launcher msg="Unrecognized bus ''"
level=error timestamp=2018-05-31T07:51:40.372017Z pos=libvirt_helper.go:78 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Defining the VM failed."
level=error timestamp=2018-05-31T07:51:40.372072Z pos=server.go:59 component=virt-launcher namespace=golden-images name=test-vm kind= uid=5417df7e-64a7-11e8-a2a8-42010af0000c reason="virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')" msg="Failed to sync vm"
level=info timestamp=2018-05-31T07:51:40.392744Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:41.391873Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:42.391845Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:43.391942Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:44.391931Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:45.391928Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:46.394196Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:47.391983Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:48.392334Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:49.392041Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:50.392155Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
level=info timestamp=2018-05-31T07:51:51.392525Z pos=monitor.go:209 component=virt-launcher msg="Still missing PID for qemu, Process qemu not found in /proc"
```
oc describe vms
```
[root@qe-shiywang-master-etcd-1 ~]# oc describe vm test-vm
Name:         test-vm
Namespace:    golden-images
Labels:       kubevirt.io/nodeName=qe-shiywang-node-registry-router-1
Annotations:  presets.virtualmachines.kubevirt.io/presets-applied=kubevirt.io/v1alpha1
API Version:  kubevirt.io/v1alpha1
Kind:         VirtualMachine
Metadata:
  Cluster Name:        
  Creation Timestamp:  2018-05-31T07:50:46Z
  Finalizers:
    foregroundDeleteVirtualMachine
  Generation:        1
  Resource Version:  60553
  Self Link:         /apis/kubevirt.io/v1alpha1/namespaces/golden-images/virtualmachines/test-vm
  UID:               5417df7e-64a7-11e8-a2a8-42010af0000c
Spec:
  Domain:
    Devices:
      Disks:
        Disk:
        Name:         mydisk
        Volume Name:  pvcvolume
    Machine:
      Type:  
    Resources:
      Requests:
        Memory:                      64M
  Termination Grace Period Seconds:  0
  Volumes:
    Name:  pvcvolume
    Persistent Volume Claim:
      Claim Name:  golden-pvc
Status:
  Conditions:
    Last Probe Time:       <nil>
    Last Transition Time:  2018-05-31T07:51:19Z
    Message:               server error. command Launcher.Sync failed: virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')
    Reason:                Synchronizing with the Domain failed.
    Status:                False
    Type:                  Synchronized
  Interfaces:
    Ip Address:  10.129.0.36
  Node Name:     qe-shiywang-node-registry-router-1
  Phase:         Scheduled
Events:
  Type     Reason              Age                From                                              Message
  ----     ------              ----               ----                                              -------
  Normal   SuccessfulCreate    3m                 virtualmachine-controller                         Created virtual machine pod virt-launcher-test-vm-782rs
  Normal   SuccessfulHandOver  3m                 virtualmachine-controller                         Pod owner ship transfered to the node virt-launcher-test-vm-782rs
  Warning  SyncFailed          3m (x5 over 3m)    virt-handler, qe-shiywang-node-registry-router-1  server error. command Launcher.Sync failed: virError(Code=0, Domain=0, Message='Missing error')
  Warning  SyncFailed          23s (x11 over 3m)  virt-handler, qe-shiywang-node-registry-router-1  server error. command Launcher.Sync failed: virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')
```

Anything wrong with my images ?

**Comment by cynepco3hahue** [comment]
It looks like the issue https://github.com/kubevirt/kubevirt/issues/1068, @slintes already has PR, so I hope soon you will have the fix

**Comment by shiywang** [comment]
same situation in gluster-fs I forgot to mention
```
[root@qe-shiywang-master-etcd-1 kubevirt-ansible]# oc describe vm test-vm
Name:         test-vm
Namespace:    golden-images
Labels:       <none>
Annotations:  presets.virtualmachines.kubevirt.io/presets-applied=kubevirt.io/v1alpha1
API Version:  kubevirt.io/v1alpha1
Kind:         VirtualMachine
Metadata:
  Cluster Name:        
  Creation Timestamp:  2018-06-06T11:31:14Z
  Finalizers:
    foregroundDeleteVirtualMachine
  Generation:        1
  Resource Version:  45952
  Self Link:         /apis/kubevirt.io/v1alpha1/namespaces/golden-images/virtualmachines/test-vm
  UID:               1ee6de0e-697d-11e8-bcfe-fa163e954c59
Spec:
  Domain:
    Devices:
      Disks:
        Disk:
        Name:         mydisk
        Volume Name:  pvcvolume
    Machine:
      Type:  
    Resources:
      Requests:
        Memory:                      64M
  Termination Grace Period Seconds:  0
  Volumes:
    Name:  pvcvolume
    Persistent Volume Claim:
      Claim Name:  golden-pvc
Status:
  Phase:  Scheduling
Events:
  Type    Reason            Age   From                       Message
  ----    ------            ----  ----                       -------
  Normal  SuccessfulCreate  37s   virtualmachine-controller  Created virtual machine pod virt-launcher-test-vm-2nh59
[root@qe-shiywang-master-etcd-1 kubevirt-ansible]# oc describe vm test-vm
Name:         test-vm
Namespace:    golden-images
Labels:       kubevirt.io/nodeName=qe-shiywang-glusterfs-registry-node-2
Annotations:  presets.virtualmachines.kubevirt.io/presets-applied=kubevirt.io/v1alpha1
API Version:  kubevirt.io/v1alpha1
Kind:         VirtualMachine
Metadata:
  Cluster Name:        
  Creation Timestamp:  2018-06-06T11:31:14Z
  Finalizers:
    foregroundDeleteVirtualMachine
  Generation:        1
  Resource Version:  46079
  Self Link:         /apis/kubevirt.io/v1alpha1/namespaces/golden-images/virtualmachines/test-vm
  UID:               1ee6de0e-697d-11e8-bcfe-fa163e954c59
Spec:
  Domain:
    Devices:
      Disks:
        Disk:
        Name:         mydisk
        Volume Name:  pvcvolume
    Machine:
      Type:  
    Resources:
      Requests:
        Memory:                      64M
  Termination Grace Period Seconds:  0
  Volumes:
    Name:  pvcvolume
    Persistent Volume Claim:
      Claim Name:  golden-pvc
Status:
  Conditions:
    Last Probe Time:       <nil>
    Last Transition Time:  2018-06-06T11:31:51Z
    Message:               server error. command Launcher.Sync failed: virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')
    Reason:                Synchronizing with the Domain failed.
    Status:                False
    Type:                  Synchronized
  Interfaces:
    Ip Address:  10.129.0.11
  Node Name:     qe-shiywang-glusterfs-registry-node-2
  Phase:         Scheduled
Events:
  Type     Reason              Age               From                                                 Message
  ----     ------              ----              ----                                                 -------
  Normal   SuccessfulCreate    40s               virtualmachine-controller                            Created virtual machine pod virt-launcher-test-vm-2nh59
  Normal   SuccessfulHandOver  2s                virtualmachine-controller                            Pod owner ship transfered to the node virt-launcher-test-vm-2nh59
  Warning  SyncFailed          0s (x10 over 3s)  virt-handler, qe-shiywang-glusterfs-registry-node-2  server error. command Launcher.Sync failed: virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')
[root@qe-shiywang-master-etcd-1 kubevirt-ansible]# 
[root@qe-shiywang-master-etcd-1 kubevirt-ansible]# oc describe vm test-vm
Name:         test-vm
Namespace:    golden-images
Labels:       kubevirt.io/nodeName=qe-shiywang-glusterfs-registry-node-2
Annotations:  presets.virtualmachines.kubevirt.io/presets-applied=kubevirt.io/v1alpha1
API Version:  kubevirt.io/v1alpha1
Kind:         VirtualMachine
Metadata:
  Cluster Name:        
  Creation Timestamp:  2018-06-06T11:31:14Z
  Finalizers:
    foregroundDeleteVirtualMachine
  Generation:        1
  Resource Version:  46079
  Self Link:         /apis/kubevirt.io/v1alpha1/namespaces/golden-images/virtualmachines/test-vm
  UID:               1ee6de0e-697d-11e8-bcfe-fa163e954c59
Spec:
  Domain:
    Devices:
      Disks:
        Disk:
        Name:         mydisk
        Volume Name:  pvcvolume
    Machine:
      Type:  
    Resources:
      Requests:
        Memory:                      64M
  Termination Grace Period Seconds:  0
  Volumes:
    Name:  pvcvolume
    Persistent Volume Claim:
      Claim Name:  golden-pvc
Status:
  Conditions:
    Last Probe Time:       <nil>
    Last Transition Time:  2018-06-06T11:31:51Z
    Message:               server error. command Launcher.Sync failed: virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')
    Reason:                Synchronizing with the Domain failed.
    Status:                False
    Type:                  Synchronized
  Interfaces:
    Ip Address:  10.129.0.11
  Node Name:     qe-shiywang-glusterfs-registry-node-2
  Phase:         Scheduled
Events:
  Type     Reason              Age                From                                                 Message
  ----     ------              ----               ----                                                 -------
  Normal   SuccessfulCreate    50s                virtualmachine-controller                            Created virtual machine pod virt-launcher-test-vm-2nh59
  Normal   SuccessfulHandOver  12s                virtualmachine-controller                            Pod owner ship transfered to the node virt-launcher-test-vm-2nh59
  Warning  SyncFailed          3s (x12 over 13s)  virt-handler, qe-shiywang-glusterfs-registry-node-2  server error. command Launcher.Sync failed: virError(Code=20, Domain=20, Message='missing target information for device /var/run/kubevirt-private/vm-disks/pvcvolume/disk.img')
```

**Comment by rmohr** [comment]
Yep fixed with #1068. I triggered a pre-release. @shiywang you can check the release-page in about half an hour from now and should find a `v0.5.1-alpha.3` release which contains this and a few other fixes. @shiywang feel free to reopen if you still experience the issue with this release.
